scripts/build.js

```diff
@@ -0,0 +1,12 @@
+var fs = require('fs');
+var path = require('path');
+
+var inputFile = path.join(__dirname, '..', 'dpdb-formatted.json');
+var outputFile = path.join(__dirname, '..', 'dpdb.json');
+var formattedJSON = fs.readFileSync(inputFile, 'utf8');
+
+// Use JSON.stringify/parse to ensure that the JSON is valid
+// rather than just stripping out space/newline chars
+var minifiedJSON = JSON.stringify(JSON.parse(formattedJSON));
```

Comment:
The original is formatted already, so makes sense for the other version to be minified, no? Maybe the names should just be dpdb.json and dpdb.min.json or something?